storage/client/pom.xml

```diff
@@ -31,6 +31,20 @@
     <tag>HEAD</tag>
   </scm>
 
+  <pluginRepositories>
```

Comment:
Just curious what the usage of the pluginRepository?

Reply:
this repo, gets the dependencies for groovy-eclipse-compiler